edx_sga/static/js/src/edx_sga.js

```diff
@@ -104,7 +105,30 @@ function StaffGradedAssignmentXBlock(runtime, element) {
             updateChangeEvent(fileUpload);
         }
 
-        function renderStaffGrading(data) {
+        // function to replace the entire content of the global variable
+        // containing all the assignments
+        function replaceStudentsAssignments (data) {
+          studentAssignments = data;
+          renderStaffGrading();
+        }
+
+        // function to update one student assignment
+        function updateStudentAssignment (data) {
+          _.map(data.assignments, function (assignment) {
```

Comment:
Just passing by, but we should use `_.each` if we're going to mutate a variable and not use the return value